creators/stops_creator.py

```diff
@@ -53,12 +77,9 @@ def _fill_stops(self, stops, route):
         if isinstance(route, Route):
             i = 0
             for stop in route.stops:
-                if stop in stops:
-                    # Replace stop id with Stop objects
-                    # TODO: Remove here and use references in TripsCreatorFenix
-                    route.stops[i] = stops[stop]
-                else:
-                    raise RuntimeError("Unknown stop: " + str(stop))
+                # Replace stop id with Stop objects
+                # TODO: Remove here and use references in TripsCreatorFenix
+                route.stops[i] = self._get_stop(stop, stops)
```

Comment:
this is a bad pattern, the object type of route.stops is inconsistent:
If you use the default creator, you have a GTFS Stop objects list, if not, you have an OSM2GTFS Stop objects list.
You may want instead to create another object, such as route.gtfs_stops ?

This breaks the Accra creator (that do not use the default creator, and need to get the stops of the routes in the trip creator).

Reply:
You are right about the inconsistency but that issue is not part of this PR. The actual "default stops_creator" implementation (already in the master branch) behaves as the fenix and incofer stops_creators, so this inconsistency is inherited from the previous stop creators.

This inconsistency should be a issue to be solve in another pull request.

Reply:
>this is a bad pattern, the object type of route.stops is inconsistent:
>If you use the default creator, you have a GTFS Stop objects list, if not, you have an OSM2GTFS Stop objects list.

After reviewing again the code ( and remembering how it works 🤔  ) I noticed that the inconsistency is not as you said. If you use the default creator you have OSM2GTFS Stop Objects list but if you don't use the default creator you will have a strings list, where each string corresponds to the node ID the corresponds to a stop in OSM.

Reply:
you are right. Ok for an issue.
Maybe it is what the TODO at the previous was supposed to mean ..

Reply:
Here is the issue #71